source/Core/ModuleList.cpp

```diff
@@ -104,6 +111,12 @@ FileSpec ModuleListProperties::GetClangModulesCachePath() const {
       ->GetCurrentValue();
 }
 
+bool ModuleListProperties::GetUseDWARFImporter() const {
+  const uint32_t idx = ePropertyUseDWARFImporter;
+  return m_collection_sp->GetPropertyAtIndexAsBoolean(
```

Comment:
you can probably fold `idx` in the caller.

Reply:
This is a pattern in LLDB. Doing it the same way as everywhere will make it easier to search&replace these accessors once we come up with a nicer templated schema.